for-newbies/60-minute-quick-start/adding-a-model.md

```diff
@@ -120,6 +120,11 @@ Open the `contacts/index.cfm` and add the following to the view:
 </cfoutput>
 ```
 
+{% hint style="warning" %}
+You need to restart the application to see the changes. Whenever you edit a model, a server restart will be needed. You can achieve it using the command `server restart` via CommandBox. Please check out the API Docs to discover CommandBox:
```

Comment:
You actually just need to reinit the framework to refresh singletons.  So this paragraph is not accurate @lvega-ortus 

I would suggest the following.

```
Note: If your models are `singletons`, they will persist for the life-span of your ColdFusion application.  If you want see code changes for singletons, then you can reinit the framework by using the `?fwreinit={password}` Url action or via CommandBox using `coldbox reinit`.
```